core/src/testUtils/groovy/nu/yona/server/test/User.groovy

```diff
@@ -0,0 +1,101 @@
+package nu.yona.server.test
+
+import nu.yona.server.YonaServer
+import groovy.json.*
+
+class User
+{
+	final String firstName
+	final String lastName
+	final String mobileNumber
+	final String mobileNumberConfirmationUrl
+	final boolean hasPrivateData
+	final String mobileNumberConfirmationCode
+	final String nickname
+	final List<String> devices
+	final List<String> goals
+	final List<Buddy> buddies
+	final VPNProfile vpnProfile
+	final String url
+	final String password
+	User(def json, String password)
+	{
+		this(json, true)
+		this.password = password
+	}
+	User(def json, String password, boolean hasPrivateData)
+	{
+		this(json, hasPrivateData)
+		this.password = password
+	}
+	User(def json)
+	{
+		this(json, false)
+	}
+	private User(def json, boolean hasPrivateData)
+	{
+		this.firstName = json.firstName
```

Comment:
Accepted. For User it indeed makes sense to optimize a little because it is so often used.